mt-batcher/services/sequencer/driver.go

```diff
@@ -775,7 +782,10 @@ func (d *Driver) RollupMainWorker() {
 	for {
 		select {
 		case <-ticker.C:
-			log.Info("MtBatcher eigen da sequencer fetching current block range")
+			if d.Cfg.MinTimeoutRollupTxn > d.Cfg.RollUpMinTxn {
+				log.Error("MtBatcher MinTimeoutRollupTxn more than RollUpMinTxn error ", "MinTimeoutRollupTxn(%v)>RollUpMinTxn(%v)", d.Cfg.MinTimeoutRollupTxn, d.Cfg.RollUpMinTxn)
```

Comment:
The validation of the log should be placed at the initialization of the service, and if it is unreasonable, then panic. Instead of repeatedly logging the same content in this loop

Reply:
Fixed. add function `GetBatchBlockRangeWithTimeout`.